catalog/rc.yaml

```diff
@@ -0,0 +1,1006 @@
+image:
```

Comment:
is this auto-generated?

Reply:
This was generated using opm migrate command to import an existing catalog

This was imported using the production Red Hat catalog, but now we have to manually maintain it.